packages/client/src/components/emoji-picker.section.vue

```diff
@@ -24,7 +24,7 @@ const props = defineProps<{
 }>();
 
 const emit = defineEmits<{
-	(e: 'chosen', v: string, ev: MouseEvent): void;
+	(event: 'chosen', v: string, ev: MouseEvent): void;
```

Comment:
I think consistent naming would be nice.
```suggestion
	(ev: 'chosen', v: string, ev: MouseEvent): void;
```

Reply:
True, I was debating with myself how to approach this the best way since `ev` is taken as an argument name here already 😵‍💫

Reply:
Decided to fix this by switching it around, using `ev` as the event name (as everywhere) and `event` for the event class type